### PR TITLE
EBODAC-511 - Changes for the last issues in Jenkins and improvement for Exceptions (05-08-2016)

### PR DESCRIPTION
**Pull request by rubenmartin**
@pmuchowski  please test it

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/helper/enumBookingAppFilters.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
+package org.motechproject.ebodac.uitest.helper;
+
+public enum enumBookingAppFilters {
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/helper/enumBookingAppFilters.java]: @rubenmartin the name should be "BookingAppFilters" remove the "enum" from the name

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java [hunk 2]
```diff
@@ -3,45 +3,62 @@
...
+
+    private static final String FORMAT_DATE = "yyyy-MM-dd";
+
+    private static final String TITLE = "title";
+    public static final String URL_PATH = "/#/bookingApp/screening";
+    static final By SCREENING_BUTTON = By.xpath("//div[@id='main-content']/div/div/div/button[1]");
     static final By SCREENING_BUTTON_NG_CLICK = By.xpath("//button[@ng-click='addScreening()']");
-    static final By DATE_FIELD = By.xpath("//div[@class='modal-body']/div/input[@type='text']");
-    static final By DAY = By.linkText("13");
+    // static final By DATE_FIELD =
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java]: you should remove commented out code

File: ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java [hunk 3]
```diff
@@ -179,54 +310,122 @@ public void setDate() throws InterruptedException {
...
 
     public boolean bookingIdExistsOnPage(String id) {
+        boolean status = false;
         try {
             By elementBookingId = By.cssSelector("td[title=\"" + id + "\"]");
             sleep(TIMEOUT);
-            waitForElement(elementBookingId);
+
+            waitForElementToBeEnabled(elementBookingId);
+            // waitForElement(elementBookingId);
```
Comment (pmuchowski) [inline ebodac/src/test/java/org/motechproject/ebodac/uitest/page/BookingAppScreeningPage.java]: you should remove commented out code

**Comment by rubenmartin** [comment]
@pmuchowski  removed all the additional non needed comments , please test

**Comment by pmuchowski** [comment]
test this please

**Pull request merged**